cmd/yaegi/run.go

```diff
@@ -56,14 +56,14 @@ func run(arg []string) error {
 	}
 
 	if cmd != "" {
-		i.REPL(strings.NewReader(cmd), os.Stderr)
+		_, err = i.Eval(cmd)
```

Comment:
Did you change that REPL call to an Eval call on purpose?

Reply:
Yes. Calling `Eval`directly here is simpler and clearer. Now we use `REPL` only in explicit interactive mode, which makes sense.